cmd/utils/flags.go

```diff
@@ -660,6 +660,31 @@ var (
 		Usage: "Default minimum difference between two consecutive block's timestamps in seconds",
 		Value: eth.DefaultConfig.Istanbul.BlockPeriod,
 	}
+
+	// Proxy node settings
+	ProxyFlag = cli.BoolFlag{
+		Name:  "proxy.proxy",
+		Usage: "Specifies whether this node is a proxy",
+	}
+	ProxyInternalFacingEndpointFlag = cli.StringFlag{
+		Name:  "proxy.internalendpoint",
+		Usage: "Specifies the internal facing endpoint for this proxy to listen to",
+		Value: ":30503",
```

Comment:
This flag's value is actually a full endpoint (e.g. <ip address>:<port>), as opposed to just a port, so that was the reasoning for the name difference.

Reply:
Are endpoints used elsewhere in geth? Just going off of the usage as a user I'd be unsure of what format for the value is expected

Reply:
It's not used for any other flag.  However, all other related flags just require a port, and this is the first one that needs both `address` and `port` (btw, I just noticed that my previous comment needed an escape for the brackets, so you probably didn't see that).

I think your point being unsure of the format in the usage makes a lot of sense.  I'll modify the usage text.